src/Raven.Server/Dashboard/DatabasesInfo.cs

```diff
@@ -21,6 +22,18 @@ public override DynamicJsonValue ToJson()
             json[nameof(Items)] = new DynamicJsonArray(Items.Select(x => x.ToJson()));
             return json;
         }
+
+        public override DynamicJsonValue ToJsonWithFilter(Func<string, bool> filter)
+        {
+            var items = new DynamicJsonArray(Items.Where(x => filter(x.Database)).Select(x => x.ToJson()));
```

Comment:
let's avoid LINQ, some of these notifications are created often when Studio is connected

Reply:
done